docs/csharp/programming-guide/arrays/passing-arrays-as-arguments.md

```diff
@@ -11,48 +11,48 @@ Arrays can be passed as arguments to method parameters. Because arrays are refer
 ## Passing Single-Dimensional Arrays As Arguments  
  You can pass an initialized single-dimensional array to a method. For example, the following statement sends an array to a print method.  
   
- [!code-csharp[csProgGuideArrays#34](../../../csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_1.cs)]  
+ [!code-csharp[csProgGuideArrays#34](codesnippet/CSharp/passing-arrays-as-arguments_1.cs)]  
   
  The following code shows a partial implementation of the print method.  
   
- [!code-csharp[csProgGuideArrays#33](../../../csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_2.cs)]  
+ [!code-csharp[csProgGuideArrays#33](codesnippet/CSharp/passing-arrays-as-arguments_2.cs)]  
   
  You can initialize and pass a new array in one step, as is shown in the following example.  
   
- [!code-csharp[CsProgGuideArrays#35](../../../csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_3.cs)]  
+ [!code-csharp[CsProgGuideArrays#35](codesnippet/CSharp/passing-arrays-as-arguments_3.cs)]  
   
 ## Example  
   
 ### Description  
- In the following example, an array of strings is initialized and passed as an argument to a `PrintArray` method for strings. The method displays the elements of the array. Next, methods `ChangeArray` and `ChangeArrayElement` are called to demonstrate that sending an array argument by value does not prevent changes to the array elements.  
+ In the following example, an array of strings is initialized and passed as an argument to a `PrintArray` method for strings. The method displays the elements of the array. Next, the `ChangeArray` method is called to reverse the array elements, and then the `ChangeArrayElement` method is called to modify the first three elements of the array. After each method returns, the `PrintArray` method is called to show that passing an array by value does not prevent changes to the array elements.  
```

Comment:
First, if you rename `PrintArray` to `DisplayArray`, don't forget to make it here as well.
Second, there is no `ChangeArrayElement` method, plural is used in the code: `ChangeArrayElements`